### PR TITLE
KAS-1841: Publications Tab

### PR DESCRIPTION
**Pull request by rafael-mindreau**
# `NEW` Publications Tab

## 🤔  What is this?
**Check out the branch** and see for yourself:
image

New tab with a completely new view for new users. A way to view all publications, and the specific workflow around it from translations, signing to eventually pre-press.

## 📦  What's in this box?
Mocked rows of handled, and to-do publications. Their flow and progress is indicated with new status pills. There's separate tabs, pagination and placeholder buttons and search fields for future stories to fill in.

## 🗒️  Note on components
We're moving towards a better design-implementation workflow, and this PR screams all of that as well. We've added many components. Many of which you could say that we could have used the existing method of implementation. For example a button would be:

```html
<button class="vl-button ..." />
```

Instead you will find:

```html
<Button @icon="..." />
```

This is new, and just **one of the new components** I've added in this PR. The story here is that we're moving to align with the prototype code as we're going. The old methods implemented WebUniversum, which we are no longer going to go for. The components will reflect the prototype as much as possible. The prototype can be found here: https://github.com/mono-company/kaleidos-ember-prototype you may need to explicitly request access.

### CODE REVIEW

**Comment by Wolfr** [comment]
@rafael-mindreau betekent dit dat we als deze PR gemerged is een beweging kunnen maken naar zoveel mogelijk `<Button />` syntax gebruiken?

**Review by woutersf — APPROVED** [review]
Nice

**Comment by rafael-mindreau** [comment]
> @rafael-mindreau betekent dit dat we als deze PR gemerged is een beweging kunnen maken naar zoveel mogelijk `<Button />` syntax gebruiken?

**Comment by rafael-mindreau** [comment]
@Wolfr Ja, eventueel. Dat zou vraagt wel een story. Lijkt geen werk, kan implicaties hebben...

File: app/pods/components/publications/publications-overview-table/template.hbs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        <th>Start publicatieflow</th>
+        <th>Uiterste publicatie</th>
+        <th>Vertalingen</th>
+        <th>Handtekeningen</th>
+        <th>Drukproef</th>
+        <th></th>
+      </tr>
+    </thead>
+    <tbody>
+      {{#each publicationsForCurrentPage as |publication|}}
```
Comment (brechtPhilips) [inline app/pods/components/publications/publications-overview-table/template.hbs:19]: zou this. publicationsForCurrentPage moeten zijn

File: app/pods/components/tab/template.hbs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<li class="vlc-tabs__tab"...attributes>
+  <LinkTo @route="{{@route}}">
```
Comment (brechtPhilips) [inline app/pods/components/tab/template.hbs:2]: Moet dit niet zonder de "?  @route=@route ?

**Review by brechtPhilips — COMMENTED** [review]
Nagekeken , 2 kleine vraagjes erin gezet.

File: app/pods/components/tab/template.hbs (hunk 2 above)
Comment (rafael-mindreau) [reply app/pods/components/tab/template.hbs:2]: @route={{@route}} is het enige wat nog gaat

**Comment by rafael-mindreau** [comment]
@brechtPhilips https://github.com/kanselarij-vlaanderen/kaleidos-frontend/pull/522 followup PR

**Pull request merged**